pom.xml

```diff
@@ -1108,6 +1108,32 @@
 				<sonar.host.url>https://sonarcloud.io</sonar.host.url>
 			</properties>
 		</profile>
+		
+		<profile>
```

Comment:
Do we still need this?

Reply:
I updated the mockito-core version for Java 17 because when using mockito-core:3.12.4 we get the following error.

`java.lang.NullPointerException: Cannot read the array length because "this.buf" is null`

mockito-core:5.6.0 does not support Java 8, so we need the profile.

Reply:
It would be great if you put that comment at the top of the profile?

Reply:
I added the comment.